clean up iptables/ipvs proxy mode descriptions a bit (#44110)
kubernetes/website: content/en/docs/reference/networking/virtual-ips.md

mechanism for {{< glossary_tooltip term_id="service" text="Services">}}
of `type` other than
[`ExternalName`](/docs/concepts/services-networking/service/#externalname).

> Kube-proxy watches the Kubernetes {{< glossary_tooltip

Reviewer: ```suggestion
Each instance of kube-proxy watches the Kubernetes {{< glossary_tooltip
```

term_id="control-plane" text="control plane" >}} for the addition and
removal of Service and EndpointSlice {{< glossary_tooltip

> term_id="object" text="objects." >}} For each Service, kube-proxy

Reviewer: ```suggestion
term_id="object" text="objects" >}}. For each Service, kube-proxy
```

Author: oh, good, I had just moved this from somewhere else and was like "wow... that's... a style choice"

calls appropriate APIs (depending on the kube-proxy mode) to configure
the node to capture traffic to the Service's `clusterIP` and `port`,

> and redirect that traffic to one of the Service's endpoint IPs

Reviewer: ```suggestion
and redirect that traffic to one of the Service's endpoints
```
It's a tuple of (address, port, &hellip;) I think, not just an IP address.

> (usually a Pod, but possibly an arbitrary user-provided IP). A control

Reviewer: ```suggestion
(usually a Pod, but possibly an arbitrary user-provided IP address). A control
```

#### Optimizing iptables mode performance

In iptables mode, kube-proxy creates a few iptables rules for every

> Service, and a few iptables rules for every endpoint IP. In

Reviewer: ```suggestion
Service, and a few iptables rules for each endpoint IP address. In
```

When kube-proxy starts in IPVS proxy mode, it verifies whether IPVS
kernel modules are available. If the IPVS kernel modules are not detected, then kube-proxy

> will exit with an error.

Reviewer: ```suggestion
exits with an error. Kubernetes does offer you some features that you can use to help
with that; for example, you can define an
{{< glossary_tooltip text="init containers" term_id="init-container" >}} that ensures the relevant
module are loaded before starting the container for `kube-proxy`.
```
?

Reviewer: Can we document the minimum Linux kernel version for IPVS mode (maybe for different flavors, if that matters)?

I'd assumed it was 2.2.0 or something venerable, that we could assume everyone has.

> will exit with an error.

Reviewer: Try this, then:
```
exits with an error.
```
